### ISSUE TITLE
.app version is being replaced by non_app version

### ISSUE THREAD

**Issue by cbenhagen** [Digital-Sapphire/PyUpdater#33]
```
2016-11-04 22:52:00,302 root                         INFO     Version: 0.1.0
2016-11-04 22:52:00,303 pyupdater.client             DEBUG    Setting up directories...
2016-11-04 22:52:00,303 pyupdater.client             DEBUG    Downloading key file
2016-11-04 22:52:00,303 pyupdater.client.downloader  DEBUG    Url for request: http://localhost:8000/keys.gz
2016-11-04 22:52:00,303 urllib3.connectionpool       DEBUG    Starting new HTTP connection (1): localhost
2016-11-04 22:52:00,308 urllib3.connectionpool       DEBUG    http://localhost:8000 "GET /keys.gz HTTP/1.1" 200 179
2016-11-04 22:52:00,308 pyupdater.client.downloader  DEBUG    Resource URL: http://localhost:8000/keys.gz
2016-11-04 22:52:00,308 pyupdater.client.downloader  DEBUG    Got content length of: 179
2016-11-04 22:52:00,309 pyupdater.client.downloader  DEBUG    Block size: 356
2016-11-04 22:52:00,309 pyupdater.client.downloader  DEBUG    {u'status': u'downloading', u'downloaded': 179, u'total': 179, u'percent_complete': u'100.0', u'time': u'--:--'}
2016-11-04 22:52:00,309 pyupdater.client.downloader  DEBUG    {u'status': u'finished', u'downloaded': 179, u'total': 179, u'percent_complete': u'100.0', u'time': u'00:00'}
2016-11-04 22:52:00,309 pyupdater.client.downloader  DEBUG    Download Complete
2016-11-04 22:52:00,309 pyupdater.client.downloader  DEBUG    No hash to verify
2016-11-04 22:52:00,309 pyupdater.client             DEBUG    Key file download successful
2016-11-04 22:52:00,310 dsdev_utils.paths            DEBUG    Changing to Directory --> /Users/ben/Library/Application Support/test_app
2016-11-04 22:52:00,310 pyupdater.client             DEBUG    Writing version file to disk
2016-11-04 22:52:00,310 dsdev_utils.paths            DEBUG    Moving back to Directory --> /
2016-11-04 22:52:00,312 pyupdater.client             DEBUG    Key file verified
2016-11-04 22:52:00,312 pyupdater.client             DEBUG    Loading version file...
2016-11-04 22:52:00,313 pyupdater.client             DEBUG    Downloading online version file
2016-11-04 22:52:00,313 pyupdater.client.downloader  DEBUG    Url for request: http://localhost:8000/versions.gz
2016-11-04 22:52:00,313 urllib3.connectionpool       DEBUG    Starting new HTTP connection (1): localhost
2016-11-04 22:52:00,314 urllib3.connectionpool       DEBUG    http://localhost:8000 "GET /versions.gz HTTP/1.1" 200 917
2016-11-04 22:52:00,315 pyupdater.client.downloader  DEBUG    Resource URL: http://localhost:8000/versions.gz
2016-11-04 22:52:00,315 pyupdater.client.downloader  DEBUG    Got content length of: 917
2016-11-04 22:52:00,315 pyupdater.client.downloader  DEBUG    Block size: 1832
2016-11-04 22:52:00,315 pyupdater.client.downloader  DEBUG    {u'status': u'downloading', u'downloaded': 917, u'total': 917, u'percent_complete': u'100.0', u'time': u'--:--'}
2016-11-04 22:52:00,315 pyupdater.client.downloader  DEBUG    {u'status': u'finished', u'downloaded': 917, u'total': 917, u'percent_complete': u'100.0', u'time': u'00:00'}
2016-11-04 22:52:00,315 pyupdater.client.downloader  DEBUG    Download Complete
2016-11-04 22:52:00,316 pyupdater.client.downloader  DEBUG    No hash to verify
2016-11-04 22:52:00,316 pyupdater.client             DEBUG    Version file download successful
2016-11-04 22:52:00,316 dsdev_utils.paths            DEBUG    Changing to Directory --> /Users/ben/Library/Application Support/test_app
2016-11-04 22:52:00,316 pyupdater.client             DEBUG    Writing version file to disk
2016-11-04 22:52:00,316 dsdev_utils.paths            DEBUG    Moving back to Directory --> /
2016-11-04 22:52:00,317 pyupdater.client             DEBUG    Data type: <type 'str'>
2016-11-04 22:52:00,317 pyupdater.client             DEBUG    Deleting signature from update data
2016-11-04 22:52:00,319 pyupdater.client             DEBUG    Version file verified
2016-11-04 22:52:00,319 pyupdater.client             DEBUG    Version Data:
{u'latest': {u'test_app': {u'stable': {u'mac': u'0.2.0.2.0'}}}, u'updates': {u'test_app': {u'0.3.0.2.0': {u'mac': {u'filename': u'test_app-mac-0.3.0.tar.gz', u'file_hash': u'cb8468b5749c708d98a62813c973959c8fa17544727eaebd7d4893be6dcb3f92', u'patch_name': u'test_app-mac-24', u'file_size': 6872134, u'patch_hash': u'42c5ee262b5357906a0db1eadc54a45e2ce43b235ee38166e7f3dc34017a0732', u'patch_size': 2200}}, u'1.0.0.2.0': {u'mac': {u'filename': u'test_app-mac-1.0.0.tar.gz', u'file_hash': u'fcb56551f7297116a01ca5ba2d69d744861c1c364b6eb453e7ae18def0720fd9', u'patch_name': u'test_app-mac-15', u'file_size': 6984789, u'patch_hash': u'96b577d575bf2bb5a9b513c917399c319f660fe760cf03dc4323531cfd87a090', u'patch_size': 223}}, u'1.0.1.2.0': {u'mac': {u'filename': u'test_app-mac-1.0.1.tar.gz', u'file_hash': u'0b0084362a8d9d7662b652ab17989a99e6ad9d4c3398ff0ce8baf3c3d07fd7ab', u'patch_name': u'test_app-mac-16', u'file_size': 6984789, u'patch_hash': u'ef74c0cfa058a202f53a7479cd3a86b36ab5500df2180aa736117a6c20ab6af9', u'patch_size': 8088}}, u'0.0.1.2.4': {u'mac': {u'filename': u'test_app-mac-0.0.1-4-g2073045.tar.gz', u'file_hash': u'bfdc578f98a6e434f619186fbdd37475338a46db1e18bead7f76760c6473de14', u'patch_name': u'test_app-mac-4', u'file_size': 6626350, u'patch_hash': u'71f7e1fa07fb90240cc62564f79e5220d14fbcf7b4092f03175e289f7ec6b780', u'patch_size': 58405}}, u'0.1.0.2.0': {u'mac': {u'file_hash': u'213666246d85363e4bf4f4c2c37db88490aa4b0074194b904ad1ca5142a21b8c', u'file_size': 6872134, u'filename': u'test_app-mac-0.1.0.tar.gz'}}, u'0.1.1.2.0': {u'mac': {u'filename': u'test_app-mac-0.1.1.tar.gz', u'file_hash': u'9f7cc385dd4e5aef69c0c1b1ebfa3b993d60c0af09e254ded9801845bc3a30e4', u'patch_name': u'test_app-mac-17', u'file_size': 6984789, u'patch_hash': u'df3a23ebc6b36e6bab0603eb36326045175f5ee471a0eee5ab9bb841bb41728e', u'patch_size': 236}}, u'0.2.0.2.0': {u'mac': {u'filename': u'test_app-mac-0.2.0.tar.gz', u'file_hash': u'2198f04f3d39eddae9b2660c8dd0b405a7b3d56d3af494a2cf4540e16d17c676', u'patch_name': u'test_app-mac-25', u'file_size': 6872134, u'patch_hash': u'b3681ebd0567d83f30d4a6e30f62f95457e8fb0ccf04c05b1a4209ff604932aa', u'patch_size': 4508}}}}}
2016-11-04 22:52:00,320 dsdev_utils.helpers          DEBUG    Version str: 0.1.0
2016-11-04 22:52:00,320 pyupdater.client             DEBUG    Checking for test_app updates...
2016-11-04 22:52:00,320 pyupdater.client.updates     DEBUG    Highest version: 0.2.0.2.0
2016-11-04 22:52:00,320 dsdev_utils.helpers          DEBUG    Version str: 0.2.0.2.0
2016-11-04 22:52:00,320 dsdev_utils.helpers          DEBUG    Setting release as stable. Disregard if not prerelease
2016-11-04 22:52:00,320 pyupdater.client             DEBUG    Current vesion: 0.1.0.2.0
2016-11-04 22:52:00,320 pyupdater.client             DEBUG    Latest version: 0.2.0.2.0
2016-11-04 22:52:00,321 pyupdater.client             DEBUG    Update Needed: True
2016-11-04 22:52:00,321 pyupdater.client.updates     DEBUG    Highest version: 0.2.0.2.0
2016-11-04 22:52:00,321 pyupdater.client.updates     DEBUG    Filename for test_app-0.1.0.2.0: test_app-mac-0.1.0.tar.gz
2016-11-04 22:52:00,321 pyupdater.client.updates     DEBUG    Filename for test_app-0.2.0.2.0: test_app-mac-0.2.0.tar.gz
2016-11-04 22:52:00,321 pyupdater.client.updates     DEBUG    Beginning removal of old updates
2016-11-04 22:52:00,321 pyupdater.package_handler.package DEBUG    Filename: test_app-mac-0.1.0.tar.gz
2016-11-04 22:52:00,321 pyupdater.package_handler.package DEBUG    Directory: /Users/ben/Library/Application Support/test_app/update
2016-11-04 22:52:00,321 dsdev_utils.helpers          DEBUG    Version str: test_app-mac-0.1.0.tar.gz
2016-11-04 22:52:00,321 dsdev_utils.helpers          DEBUG    Removed ".tar.gz"
2016-11-04 22:52:00,322 pyupdater.package_handler.package DEBUG    /Users/ben/Library/Application Support/test_app/update/test_app-mac-0.1.0.tar.gz does not exists
2016-11-04 22:52:00,322 pyupdater.package_handler.package DEBUG    Not an archive format: None
2016-11-04 22:52:00,322 dsdev_utils.paths            DEBUG    Changing to Directory --> /Users/ben/Library/Application Support/test_app/update
2016-11-04 22:52:00,370 pyupdater.client.downloader  DEBUG    Hash for binary data: 2198f04f3d39eddae9b2660c8dd0b405a7b3d56d3af494a2cf4540e16d17c676
2016-11-04 22:52:00,371 dsdev_utils.paths            DEBUG    Moving back to Directory --> /
2016-11-04 22:52:00,371 dsdev_utils.paths            DEBUG    Changing to Directory --> /Users/ben/Library/Application Support/test_app/update
2016-11-04 22:52:00,400 pyupdater.client.downloader  DEBUG    Hash for binary data: 2198f04f3d39eddae9b2660c8dd0b405a7b3d56d3af494a2cf4540e16d17c676
2016-11-04 22:52:00,400 pyupdater.client.updates     DEBUG    Extracting Update
2016-11-04 22:52:00,454 dsdev_utils.paths            DEBUG    Moving back to Directory --> /
2016-11-04 22:52:00,455 pyupdater.client.updates     DEBUG    Looks like we're dealing with a Mac Gui
2016-11-04 22:52:00,455 pyupdater.client.updates     DEBUG    Update Location:
/Users/ben/Library/Application Support/test_app/update
2016-11-04 22:52:00,455 pyupdater.client.updates     DEBUG    Update Name: test_app
2016-11-04 22:52:00,455 pyupdater.client.updates     DEBUG    Current App location:

/Users/ben/Projects/test_app/pyu-data/files/test_app.app
2016-11-04 22:52:00,456 pyupdater.client.updates     DEBUG    Moving app to new location:

/Users/ben/Projects/test_app/pyu-data/files
2016-11-04 22:52:00,457 pyupdater.client.updates     DEBUG    Restarting
2016-11-04 22:52:00,457 pyupdater.client.updates     DEBUG    Current App: /Users/ben/Projects/test_app/pyu-data/files/test_app

```

**Comment by cbenhagen** [comment]
"Fixed" it by renaming it mac.app again in the .spec file.

``` python
app = BUNDLE(
    exe,
    name='test_app.app',
```

to

``` python
app = BUNDLE(
    exe,
    name='mac.app',
```

If this is expected behaviour we should document and catch it.

**Comment by JMSwag** [comment]
@cbenhagen Please list the steps to reproduce.

**Comment by JMSwag** [comment]
@cbenhagen how was the spec file created initially?

**Comment by cbenhagen** [comment]
It was initially created by PyUpdater. I changed the name manually which did not end well. I think we should put that in the documentation.

**Comment by JMSwag** [comment]
I agree.

**Comment by JMSwag** [comment]
Fixed in 1115379012e9b2f606c3360f8b2326bae7b26b18.
